app/src/main/java/com/tristanwiley/chatse/chat/adapters/RoomAdapter.kt

```diff
@@ -42,15 +58,16 @@ class RoomAdapter(val site: String, val list: MutableList<Room>, val context: Co
     class ListRowHolder(private val mContext: Context, itemView: View, val site: String) : RecyclerView.ViewHolder(itemView) {
         val name: TextView = itemView.findViewById(R.id.room_name)
 
-        fun bindMessage(room: Room) {
+        fun bindMessage(room: Room, roomClickListener: OnItemClickListener) {
 
             //Set the text to the itemView TextView
             name.text = room.name
 
             //OnClick, load the chat fragment
             itemView.setOnClickListener {
                 val roomNum = room.roomID.toInt()
-                (mContext as ChatActivity).loadChatFragment(ChatRoom(site, roomNum))
+                roomClickListener.onItemClick(ChatRoom(site, roomNum))
+               // (mContext as ChatActivity).loadChatFragment(ChatRoom(site, roomNum))
```

Comment:
Don't commit commented code unless there is a reason for it. If there is a reason, it should be explained.